lesson_05/types/types_app/src/main/java/com/codedifferently/lesson5/dataprovider/KevinmasonProvider.java

```diff
@@ -0,0 +1,20 @@
+package com.codedifferently.lesson5.dataprovider;
+
+import java.util.Map;
+
+public class KevinmasonProvider extends DataProvider {
```

Comment:
You're missing the @Service annotation that will cause this file to get picked up by the tests. Please add and fix any issues you find.